llvm/test/MC/Disassembler/AMDGPU/gfx11_dasm_vop3.txt

```diff
@@ -1833,6 +1839,12 @@
 # GFX11: v_dot2_f16_f16 v255, -|0xfe0b|, -|vcc_hi|, null op_sel:[0,0,0,1] ; encoding: [0xff,0x43,0x66,0xd6,0xff,0xd6,0xf0,0x61,0x0b,0xfe,0x00,0x00]
 0xff,0x43,0x66,0xd6,0xff,0xd6,0xf0,0x61,0x0b,0xfe,0x00,0x00
 
+# GFX11: v_dot2_f16_f16 v2, v0, 0x20004000, v2 ; encoding: [0x02,0x00,0x66,0xd6,0x00,0xff,0x09,0x04,0x00,0x40,0x00,0x20]
+0x02,0x00,0x66,0xd6,0x00,0xff,0x09,0x04,0x00,0x40,0x00,0x20
+
+# GFX11: v_dot2_f16_f16 v2, 0x20004000, v0, v2 ; encoding: [0x02,0x00,0x66,0xd6,0xff,0x00,0x0a,0x04,0x00,0x40,0x00,0x20]
+0x02,0x00,0x66,0xd6,0xff,0x00,0x0a,0x04,0x00,0x40,0x00,0x20
```

Comment:
How will it look like if `0x3f803f80` which represents `<bfloat 1.0, bfloat 1.0>`?

Reply:
Exactly like this: 0x3f803f80
```
v_dot2_f16_f16 v2, v0, 0x3f803f80, v2   ; encoding: [0x02,0x00,0x66,0xd6,0x00,0xff,0x09,0x04,0x80,0x3f,0x80,0x3f]
```

Reply:
I.e. there is no difference if that is 0x3f803f80 or 0x20004000. This is not an inline constant that way or another.

Reply:
1.0 is inline literal right? does it also apply to packed operand?

Reply:
1.0 is inline, but <1.0, 1.0> is not. You could use another form with inline 1.0 and op_sel_hi, but that is another instruction.

Reply:
And then `v_dot2_bf16_bf16 v2, v0, 0x3f80, v2` gives inline 1.0:
```
v_dot2_bf16_bf16 v2, v0, 1.0, v2        ; encoding: [0x02,0x00,0x67,0xd6,0x00,0xe5,0x09,0x04]
```